webwhatsapp.py

```diff
@@ -1,21 +1,42 @@
 import time
+import os
+import sys
 
 from selenium import webdriver
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support import expected_conditions as ec
 from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.chrome.options import Options
+
 
 from message import Message, MessageGroup
 from chat import Chat
 from wapi_js_wrapper import WapiJsWrapper
 from consts import Selectors, URL
 
 
+
+
 class WhatsAPIDriver(object):
-    def __init__(self, username="API"):
-        self._driver = webdriver.Firefox()
-        self.wapi_functions = WapiJsWrapper(self._driver)
+    def __init__(self, browser='firefox', username="API"):
```

Comment:
Save browser type to a member: `self.browser_type = browser`

Reply:
Code changed as requested.